include/ebpf_windows.h

```diff
@@ -49,3 +61,44 @@ typedef enum _ebpf_helper_function
     EBPF_UPDATE_ELEMENT = 2, ///< Update map element.
     EBPF_DELETE_ELEMENT = 3, ///< Delete a map element.
 } ebpf_helper_function_t;
+
+#define EBPF_MAX_GENERAL_HELPER_FUNCTION 0xFFFF
+
+// Version 1 of the eBPF extension data structures and their lengths.
+#define EBPF_ATTACH_CLIENT_DATA_CURRENT_VERSION 1
```

Comment:
question: what is the guidance when there is a breaking change? will all these versions be bumped to `2` or only versions of those structs which had a breaking change will be bumped to 2 and remaining will remain at `1`

Reply:
Only those with breaking changes.